packages/gitbook-docs/src/parseContent.ts

```diff
@@ -56,6 +56,7 @@ const schema: ConfigType = {
     swagger,
     code,
     embed,
+    'figma-embed': embed,
```

Comment:
I think this should work properly. You can add a test similar to the one of [embed](https://github.com/pagopa/developer-portal/blob/main/packages/gitbook-docs/src/__tests__/parseContent.test.ts#L613-L645). If it works you can remove the [replaceAll at line 103](https://github.com/pagopa/developer-portal/pull/315/files#diff-6168499bb600557396e98d78cabf1deda878c9ed68e7a18ad25a0ab947a00eb8R103)

```suggestion
    '@figma/embed': embed,
```

Reply:
I added the replaceAll specifically because `@figma/embed` wasn't working